mlrun/feature_store/api.py

```diff
@@ -373,6 +373,8 @@ def ingest(
             "feature set and source must be specified"
         )
 
+    # expected to enter here when running on server side because we are always creating run_config
+    # note that the above comment doesn't mean that we are not entering this on client side.
```

Comment:
TBH, this comment is a bit confusing 😄 
Why not something simple:
`# This flow may happen both on client side (user provides run config) and server side (through the ingest API)`